app/assets/javascripts/live_entry.js

```diff
@@ -357,41 +352,26 @@
                         $('#js-time-spent').html( response.timeInAid );
                     } else {
                         // If success == false, this means the bib number lookup failed, but we still need to capture the data
-                        
-                        $( '#js-live-bib' ).val( 'false' );
-                        $( '#js-effort-name' ).html( 'n/a' );
-                        $( '#js-effort-last-reported' ).html( 'n/a' );
-                        $('#js-last-reported').html( 'n/a' );
-                        $('#js-time-spent').html( 'n/a' );
+
+                        $('#js-live-bib').val('false');
+                        $('#js-effort-name').html('n/a');
+                        $('#js-effort-last-reported').html('n/a');
+                        $('#js-last-reported').html('n/a');
+                        $('#js-time-spent').html('n/a');
                     }
 
-                    $( '#js-time-in' )
-                        .removeClass( 'exists null bad good questionable' )
-                        .addClass( response.timeInExists ? 'exists' : '' )
-                        .addClass( response.timeInStatus );
-                    $( '#js-time-out' )
-                        .removeClass( 'exists null bad good questionable' )
-                        .addClass( response.timeOutExists ? 'exists' : '' )
-                        .addClass( response.timeOutStatus );
+                    $('#js-time-in')
+                        .removeClass('exists null bad good questionable')
+                        .addClass(response.timeInExists ? 'exists' : '')
+                        .addClass(response.timeInStatus);
+                    $('#js-time-out')
+                        .removeClass('exists null bad good questionable')
+                        .addClass(response.timeOutExists ? 'exists' : '')
+                        .addClass(response.timeOutStatus);
                 });
             },
 
             /**
-             * Disables or enables fields for the effort lookup form
-             *
-             * @param bool    True to enable, false to disable
-             */
-            toggleFields: function (enable) {
```

Comment:
This method would disable or enable the fields for the live entry form. I think this may still be needed.